extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -475,6 +475,15 @@ public void setSource(Source source) {
         @ConfigItem(defaultValue = "false")
         public boolean cookieForceSecure;
 
+        /**
+         * Cookie name qualifier.
+         * For example, a session cookie name for the default OIDC tenant is 'q_session' but can be changed to 'q_session_test'
+         * if
+         * this property is set to 'test'.
+         */
+        @ConfigItem
+        public Optional<String> cookieQualifier = Optional.empty();
```

Comment:
Maybe `cookieSuffix` would be more intuitive? WDYT?

Reply:
@gastaldi Yeah, I've been wondering if the `qualifier` is a bit CDI-ish 😬.  Sure, will change it, for the default cookie it will still be `q_session_test` but for a specific tenant it will become `q_session_keycloak_test` which is different to how I described it in the PR description but is better

Reply:
@gastaldi Renamed and updated  the PR description